src/notification.scss

```diff
@@ -29,315 +31,235 @@ $block: #{$fd-namespace}-notification;
-    border-left: none;
-    border-right: 0.375rem solid;
-    border-right-color: $fd-notification-indicator-border-color-default;
+    padding-right: 1rem;
+    padding-left: 0.5rem;
   }
 
-  // ------------------------ HEADER ------------------------
-  &__header {
+  &__body {
     @include fd-reset();
 
     display: flex;
-    box-shadow: inset 0 -0.0625rem $fd-notification-header-border-color;
+    justify-content: space-between;
+
+    &--message {
+      padding-top: 1rem;
+    }
   }
 
-  &__indicator {
+  &__group {
     @include fd-reset();
 
-    &--error {
-      @include fd-priority-indicator('error');
+    overflow: hidden;
 
-      color: $fd-notification-indicator-color-error;
-    }
+    .#{$block}__body {
+      padding: 1rem 0.5rem 1rem 1rem;
+      border-bottom: $fd-notification-border;
 
-    &--success {
-      @include fd-priority-indicator('success');
+      &:last-child {
+        border-bottom: none;
+      }
 
-      color: $fd-notification-indicator-color-success;
+      @include fd-rtl() {
+        padding-left: 0.5rem;
+        padding-right: 1rem;
+      }
     }
 
-    &--warning {
-      @include fd-priority-indicator('warning');
-
-      color: $fd-notification-indicator-color-warning;
+    &--tabs {
+      box-shadow: $fd-notification-box-shadow;
     }
 
-    &--information {
-      @include fd-priority-indicator('information');
+    .#{$block}__group-header {
+      @include fd-reset();
 
-      color: $fd-notification-indicator-color-information;
-    }
-  }
-
-  &__title {
-    @include fd-reset();
-
-    color: $fd-notification-title-color;
-    padding: 1rem 0;
-    margin-right: 1rem;
-    font-weight: 700;
-    width: 100%;
-
-    @include fd-rtl() {
-      margin-right: 0;
-      margin-left: 1rem;
-    }
-
-    &:first-child {
-      padding-left: 1rem;
+      display: flex;
+      align-items: baseline;
+      border-bottom: $fd-notification-border;
+      padding: 0.75rem 0.5rem 0.25rem 1rem;
 
       @include fd-rtl() {
-        padding-left: 0;
+        padding-left: 0.5rem;
         padding-right: 1rem;
       }
     }
-  }
-
-  &__close {
-    position: relative;
-    top: 0.45rem;
-    right: 0.45rem;
 
     @include fd-rtl() {
-      right: -0.45rem;
+      padding-left: 0;
+      padding-right: 0;
     }
   }
 
-  // ------------------------ BODY ------------------------
-  &__body {
+  &__content {
     @include fd-reset();
 
-    display: flex;
-    align-items: flex-end;
     width: 100%;
-  }
+    margin-left: 0.75rem;
+    overflow: auto;
 
-  &__content {
-    @include fd-reset();
+    @include fd-rtl() {
+      margin-right: 0.75rem;
+      margin-left: 0;
+    }
 
-    display: flex;
-    padding: 1rem;
-    width: 60%;
-  }
+    &:first-child {
+      margin-left: 0;
 
-  &__avatar {
-    @include fd-reset();
+      @include fd-rtl() {
+        margin-right: 0;
+      }
+    }
 
-    margin-right: 1rem;
-    min-width: 2rem;
-    min-height: 2rem;
+    .#{$block}__title {
```

Comment:
Title should have 0.25rem margin-bottom
![image](https://user-images.githubusercontent.com/26483208/93186972-88e60e00-f73f-11ea-9d68-72583edbf2e9.png)
![image](https://user-images.githubusercontent.com/26483208/93186990-8f748580-f73f-11ea-934a-cfdffc0b92cc.png)